x-pack/plugins/maps/server/mvt/mvt_routes.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import rison from 'rison-node';
+import { schema } from '@kbn/config-schema';
+import { Logger } from 'src/core/server';
+import { IRouter } from 'src/core/server';
+import { MVT_GETTILE_API_PATH, API_ROOT_PATH } from '../../common/constants';
+import { getTile } from './get_tile';
+
+const CACHE_TIMEOUT = 0; // Todo. determine good value. Unsure about full-implications (e.g. wrt. time-based data).
+
+export function initMVTRoutes({ router, logger }: { logger: Logger; router: IRouter }) {
+  router.get(
+    {
+      path: `${API_ROOT_PATH}/${MVT_GETTILE_API_PATH}`,
+      validate: {
+        query: schema.object({
+          x: schema.number(),
+          y: schema.number(),
+          z: schema.number(),
+          geometryFieldName: schema.string(),
+          fields: schema.string(),
+          requestBody: schema.string(),
+          indexPattern: schema.string(),
```

Comment:
indexPattern is a Kibana specific concept. The API does not even use the concept of indexPatterns.  I think since this API is really an elasticsearch API, this parameter should just be called `index` so its more similar to elasticsearch APIs.